bazel/foreign_cc/vpp_vcl.patch

```diff
@@ -70,15 +70,15 @@ index 8415c28fb..a017d9fc6 100755
 +
  import ply.lex as lex
  import ply.yacc as yacc
-
+ 
 diff --git src/vcl/CMakeLists.txt src/vcl/CMakeLists.txt
-index e6d8f98ff..600ef4d5f 100644
+index 610b422d1..c5e6f8ca8 100644
 --- src/vcl/CMakeLists.txt
 +++ src/vcl/CMakeLists.txt
-@@ -30,6 +30,8 @@ add_vpp_library(vppcom
-   api_headers
- )
```

Comment:
Was this removed on purpose?

Reply:
The same as 'endif(LDP_HAS_GNU_SOURCE)', just because the diff is rebase, and the
real diff is:  ;-)

diff --git src/vcl/CMakeLists.txt src/vcl/CMakeLists.txt
index 610b422d1..c5e6f8ca8 100644
--- src/vcl/CMakeLists.txt
+++ src/vcl/CMakeLists.txt
@@ -35,6 +35,8 @@ if (LDP_HAS_GNU_SOURCE)
   add_compile_definitions(HAVE_GNU_SOURCE)
 endif(LDP_HAS_GNU_SOURCE)
 
+file(COPY vppcom.h DESTINATION ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+
 add_vpp_library(vcl_ldpreload
   SOURCES
   ldp_socket_wrapper.c

Reply:
Yeah, realized I confused myself after your previous reply. Apologies for the noise